FubarDev.FtpServer.FileSystem.DotNet/DotNetFileSystem.cs

```diff
@@ -171,9 +176,18 @@ public Task<Stream> OpenReadAsync(IUnixFileEntry fileEntry, long startPosition,
         {
             var targetEntry = (DotNetDirectoryEntry)targetDirectory;
             var fileInfo = new FileInfo(Path.Combine(targetEntry.Info.FullName, fileName));
-            using (var output = fileInfo.Create())
+            try
             {
-                await data.CopyToAsync(output, _streamBufferSize, cancellationToken);
+                using (var output = fileInfo.Create())
+                {
+                    await data.CopyToAsync(output, _streamBufferSize, cancellationToken);
+                }
+            }
+            catch (OperationCanceledException)
```

Comment:
Přidat logování

Reply:
takhle to resi oni :
catch (OperationCanceledException)
                {
                    // Ignore the OperationCanceledException
                    // This is normal during disconnects
                }

Reply:
ok

Reply:
pokud se to loguje viz dolní koment, tak ok